src/cmds/scripts/pbs_habitat.in

```diff
@@ -524,6 +524,15 @@ upgrade_pbs_schema() {
 			return $ret
 		fi
 		ver="1.3.0"
+	fi
+	
+	if [ "$ver" = "1.3.0" ]; then
+		upgrade_pbs_schema_from_v1_3_0
+		ret=$?
+		if [ $ret -ne 0 ]; then
+			return $ret
+		fi
+		ver="1.4.0"
```

Comment:
Just a thought can we make this to "1.3.1" itself. Because the current database changes and the previous one are under the same release cycle.

Reply:
reverted it back to 1.3.0